README.md

```diff
@@ -118,7 +153,13 @@ entities (no need to draw an ERD)...
 Your answer...
 
 ```
-Replace this with your answer
+An erd is a outline that is made before making the database. It helps
```

Comment:
:+1: Yes but, specifically, it outlines the exact data values we can expect to have in our application and the relationships between groups of data.